python/setup.py

```diff
@@ -7,7 +7,7 @@
 setuptools.setup(
     name='dbl-tempo',
     version='0.1.0',
-    author='Ricardo Portilla, Databricks',
+    author='Ricardo Portilla, Tristan Nixon, Max Thone, Sonali Guleria',
     author_email='ricardo.portilla@databricks.com',
```

Comment:
email probably should be labs@databricks.com

Reply:
Shouldn't the issues be routed directly to the primary authors? This seems like routing to labs@ would add a lot of noise and likely be missed by tempo authors.